### DISCUSSION TITLE
Container Image documentation recommends disabling the scanners when looking just for image config, but the CLI does not allow it

### DISCUSSION THREAD

**Discussion by julianvilas** [aquasecurity/trivy#5168] [Documentation]
### Description

The Container Image documentation recommends disabling the scanners when you just want to scan the image config for secrets:
```
$ trivy image --scanners none --image-config-scanners secret vuln-image
```

That value is not allowed for the `--scanners` flag in the CLI. Example:
```
$ docker run -it --rm aquasec/trivy:0.45.0 image --scanners none --image-config-scanners secret alpine
Scan a container image

Usage:
  trivy image [flags] IMAGE_NAME
...
2023-09-12T15:09:42.131Z	FATAL	invalid argument "none" for "--scanners" flag: must be one of ["vuln" "config" "secret" "license"]
```

### Link

 https://aquasecurity.github.io/trivy/v0.45/docs/target/container_image/

https://github.com/aquasecurity/trivy/blob/cdab67e7fa35b389b8a4252f20fa6e8155f8bccb/docs/docs/target/container_image.md?plain=1#L174-L179

### Suggestions

Either removing this recommendation from the docs or adding the feature to the CLI.

**Comment by knqyf263** [comment]
Thanks. Created https://github.com/aquasecurity/trivy/issues/5182 for tracking.

**Comment by y4ney** [comment]
@knqyf263 ,The Container Image documentation has not been updated yet, see it  https://aquasecurity.github.io/trivy/v0.45/docs/target/container_image/

**Reply by nikpivkin** [reply]
Hi @y4ney . The release with this fix has not been released yet.

**Reply by y4ney** [reply]
Okay，Thanks
